overseer.yaml

```diff
@@ -154,10 +154,16 @@ components:
     Token:
       description: Authorization token
       required:
-        - Token
+        - token
+        - role
       properties:
         token:
           type: string
+        role:
+          type: string
+          enum:
```

Comment:
W sumie tak sobie mysle, ze to jets podatne na Man-in-the-Middle attack. Ale i tak nie bedzie dostepu do wszytskich zasobów jak nawet ktos oszuka response przy logowaniu.